dev.overlayed.Overlayed.yaml

```
@@ -0,0 +1,51 @@
+id: dev.overlayed.Overlayed
+
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+
+rename-icon: overlayed
+command: overlayed
+finish-args:
+  # NOTE: not sure if this is for both IPC and Websockets transports
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create # allows for discord rpc
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+
+modules:
+  - name: overlayed
+    buildsystem: simple
+    build-commands:
+      - chmod +x Overlayed.AppImage
+      - ./Overlayed.AppImage --appimage-extract
+
+      - |
+        desktop-file-edit \
+          --set-name="Overlayed App" \
+          --set-comment="Overlayed's game launcher" \
+          --set-icon="$FLATPAK_ID" \
+          --set-key="StartupNotify" --set-value="true" \
+          --set-key="Categories" --set-value="Game;ActionGame;AdventureGame;Simulation;" \
+          --set-key="Keywords" --set-value="game;minecraft;mc;" \
+          --set-key="StartupWMClass" --set-value="Overlayed" \
+          squashfs-root/usr/share/applications/overlayed.desktop
+
+      - mkdir -p /app/{bin,share}
+      - cp -r squashfs-root/usr/share/icons /app/share/icons
+      - install -Dm755 squashfs-root/usr/bin/overlayed /app/bin/overlayed
+      - install -Dm644 squashfs-root/usr/share/applications/overlayed.desktop /app/share/applications/dev.overlayed.Overlayed.desktop
+      - install -Dm644 dev.overlayed.Overlayed.metainfo.xml /app/share/metainfo/dev.overlayed.OverlayedApp.metainfo.xml
+
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/overlayeddev/overlayed/main/distribution/dev.overlayed.Overlayed.metainfo.xml
```

```suggestion
        url: https://raw.githubusercontent.com/overlayeddev/overlayed/14a6d2c8b022f7d44de4a57350e8f30ff35ef7dc/distribution/dev.overlayed.Overlayed.metainfo.xml
```

Don't use a branch here otherwise the checksum will change whenever you update causing the build to fail.